accounts/tests.py

```diff
@@ -154,49 +158,143 @@ def test_failure_post_with_mismatch_password(self):
         self.assertFalse(User.objects.filter(username='test', email='test@example.com').exists())
         self.assertFormError(response, 'form', 'password2', '確認用パスワードが一致しません。')
 
+
 class TestHomeView(TestCase):
+    def setUp(self):
+        post = {
+            'email' : 'test@example.com',
+            'username' : 'test',
+            'password1' : 'goodpass',
+            'password2' : 'goodpass',
+        }
+        self.client.post(reverse('accounts:signup'), post)
+
     def test_success_get(self):
         response = self.client.get(reverse('accounts:home'))
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'accounts/home.html')
 
 
 class TestLoginView(TestCase):
+    def setUp(self):
+        post = {
+            'email' : 'test@example.com',
+            'username' : 'test',
+            'password1' : 'goodpass',
+            'password2' : 'goodpass',
+        }
+        self.client.post(reverse('accounts:signup'), post)
+        self.client.logout()
+
     def test_success_get(self):
-        pass
+        response = self.client.get(reverse('accounts:login'))
+        self.assertEqual(response.status_code,200)

```

Comment:
```suggestion
        self.assertEqual(response.status_code, 200)
```
引数同士はスペース空けてください。他のところも同様にスペース空けてください。

Reply:
修正しました。